pyEpiabm/pyEpiabm/sweep/spatial_sweep.py

```diff
@@ -12,6 +12,7 @@
 from pyEpiabm.property import InfectionStatus
 from pyEpiabm.routine import SpatialInfection
 from pyEpiabm.utility import DistanceFunctions
+from pyEpiabm.utility import SpatialKernel as SK
```

Comment:
Is this really necessary? We don't use this syntax anywhere else, and PEP is dubious on it...

Reply:
(If it is to shorten L214, just calculate the distance in a separate line!)